accepted/2020/net5/net5.md

```diff
@@ -0,0 +1,854 @@
+`netX.Y-windowsA.B` | `NETCOREAPP`, `NET`, `NETX_Y`, `WINDOWS`, `WINDOWSA_B`
+
+Specifically:
+
+* We continue to define `NETCOREAPP` for backwards compatibility with existing
+  `#if` code.
+* Moving forward we'll use `NET` as the versionless symbol (for .NET Framework
+  we used `NETFRAMEWORK`, so no conflict there).
+* We'll follow the rule-based creation in SDK-style projects which takes the
+  friendly TFM name without the OS flavor, makes it upper-case and replaces
+  special characters with an underscore.
+* For OS flavors we'll do the same, i.e. create versionless as well as version
+  specific symbols.
+
+In order to make it easier to update code, especially when doing
+multi-targeting, we should make them additive, so that when targeting `net6.0`
+both `NET6_0` and `NET5_0` are defined. The same applies to OS bindings.
+Examples:
+
+* `net5.0`
+  * `NETCOREAPP`, `NETCOREAPP3_1` (for backwards compatibility)
+  * `NET`, `NET5_0`  
+* `net6.0`
+  * (same as `net5.0`)
+  * `NET6_0`  
+* `net5.0-ios13.0`
+  * (same as `net5.0`)
+  * `IOS`, `IOS13_0`
+* `net5.0-ios14.0`
+  * (same as `net5.0-ios13`)
+  * `IOS14_0`
+
+### What would we target?
+
+Everything that is universal or portable to many platforms will target `net5.0`.
+This includes most libraries but also ASP.NET Core and EF.
+
+Platform-specific libraries would target platform-specific flavors. For example,
+WinForms and WPF controls would target `net5.0-windows`.
+
+Cross-platform application models (Xamarin Forms, ASP.NET Core) and bridge packs
+(Xamarin Essentials) would at least target `net5.0` but might also additionally
+target platform-specific flavors to light-up more APIs or features. This
+approach is also called bait & switch.
+
+### TFMs in the UI
+
+There are some places in the IDE where targeting information is displayed:
+
+![](pic03.png)
+
+![](pic04.png)
+
+![](pic05.png)
+
+![](pic06.png)
+
+| Rule                                                                                                        | Affected UI                                                        |
+|-------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------|
+| For most UI, we should use the TFM friendly name (for example, `netcoreapp3.1`, `net5.0`, or `net5.0-ios`). | Solution Explorer, Editor Context Switcher, Debug Context Switcher |
+| For cases where we use branded display names, we should use the name .NET 5.                                | Project Properties                                                 |
+
+### Related work
+
+To support multi-targeting end-to-end the TFM unification isn't enough. We also
+need to unify the SDKs between .NET Core, Windows desktop, Xamarin, and
+maybe ASP.NET Core. Without it, multi-targeting doesn't work without having to use
+3rd party tools, such as Oren's
```

Comment:
```suggestion
3rd party tools, such as Claire's
```

Reply:
Who is **Claire**?

Reply:
@Nirmal4G She changed her name from Oren to Claire. I tremendously respect her courage and transparency.

Reply:
Do I hear trans rights?

Reply:
@Happypig375 Doesn't matter what you hear as long as what you say is gracious.

Reply:
I've seen/heard Oren's talks, pardon my ignorance, but I thought...🤯 Man, the world is **VERY** small!

_Good for ~~him~~ her !_

Reply:
@Happypig375 I can't tell whether your comment is meant to be funny or snarky. Just know that we have zero tolerance for transphobic behavior or any other violations of our [code of conduct](https://dotnetfoundation.org/code-of-conduct). As @jnm2 said, Claire deserves a tremendous amount of respect for her courage of sharing her story. We should be supportive as members of the .NET community and fellow human beings. I've marked your comment as off topic.

Reply:
> Who is **Claire**?

Just this morning, I was looking at `MSBuild.Sdk.Extras` repo, reading ~~his~~ her blogs about Reference assemblies... It was not yet changed then. I might be the last person and the first to witness that!

Reply:
Wait, that's a real person!?